wdl/import.wdl

```diff
@@ -627,20 +843,50 @@ workflow import_pheweb {
 	 	 }
 	}
 
-        call matrix { input:
-	              sites=annotation.sites_list ,
-		      pheno_gz=pheno.pheno_gz,
-		      manhattan=pheno.pheno_manhattan,
-		      bed_file = bed_file,
-		      docker=docker,
-	              mem = mem ,
-                      disk=disk ,
-	              output_url = output_url
+        # call matrix { input:
```

Comment:
We should aim to make this generic so we can also setup old fashioned browsers with all variants in for some use cases and still have only one import script.

Reply:
Right, same thing as with dao… 
I updated the workflow, what do you think? There is a boolean input parameter with which the mode for imports can be specified.